pytest.ini

```
@@ -0,0 +1,2 @@
+[pytest]
+norecursedirs = .* lif_meanfield_tools example *.egg dist build
```

I think, I don't really understand what it does: Does it tell pytest which directories should not recurse into?

If yes one could add:
`readme_figures`

and maybe change
```suggestion
norecursedirs = .* lif_meanfield_tools example *.egg* dist build
```
to cover `.egg-info`, which exists in my case.

---

Yes, you're right. These are the paths that are excluded during test collection. `.egg-info` is already excluded due to the `.*` expression.  I will add `readme_figures` though.